Email-validation/script.js

```diff
@@ -0,0 +1,24 @@
+
+
+function EmailValidation (Email)
```

Comment:
this function has been copied from another website, I need a new one that was written by you, please review these comments and fix it !!!!

Reply:
hey elias 
Yes, I wrote to you that I copied it from the Internet because I didn't have time to solve the homework, that day you uploaded the homework in same day of homework lift .
I'm going to work hard to fix it.  
thank you

Reply:
Good morning Elias 

I've adjusted and resolve the Email validation .

thank you